package.json

```diff
@@ -1,14 +1,42 @@
 {
   "name": "gridicons",
-  "version": "0.1.0",
+  "version": "1.0.0-alpha.2",
+  "main": "build/index.js",
+  "scripts": {
+    "build": "grunt --verbose",
+    "prepublish": "npm run build"
+  },
+  "peerDependencies": {
+    "react": ">=14.x"
```

Comment:
I forget how it works with peer dependencies, but should we care that this package warns on install?

```
npm WARN gridicons@1.0.0-alpha.2 requires a peer of react@>=14.x but none was installed.
```